Manganis can't pass numeric value to kotlin function

**Problem**

If i want to pass an numeric value (e.g. i64) to a kotlin function im getting this error:
```
error[E0308]: mismatched types
 --> packages/haptics/src/android.rs:8:5
  |
8 |     #[manganis::ffi("android")]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `JValueGen<&JObject<'_>>`, found `JValueGen<&&JObject<'_>>`
  |
  = note: expected enum `JValueGen<&JObject<'_>>`
             found enum `JValueGen<&&JObject<'_>>`
  = note: this error originates in the attribute macro `manganis::ffi`
```

**Steps To Reproduce**

Steps to reproduce the behavior:

Rust code example:
```rust
mod ffi {
    #[cfg(target_os = "android")]
    #[manganis::ffi("android")]
    extern "Kotlin" {
        pub type HapticsPlugin;

        pub fn vibrate(this: &HapticsPlugin, duration: i64);
    }
}
```

Kotlin:
```kotlin
class HapticsPlugin(activity: Activity) {
    private val appContext = activity.applicationContext

    private val vibrator: Vibrator = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
        val vibratorManager =
            appContext.getSystemService(Context.VIBRATOR_MANAGER_SERVICE) as VibratorManager
        vibratorManager.defaultVibrator
    } else {
        @Suppress("DEPRECATION")
        appContext.getSystemService(Context.VIBRATOR_SERVICE) as Vibrator
    }

    fun vibrate(duration: Long) {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            vibrator.vibrate(VibrationEffect.createOneShot(duration, VibrationEffect.DEFAULT_AMPLITUDE))
        } else {
            @Suppress("DEPRECATION")
            vibrator.vibrate(duration)
        }
    }
}
```

Can be simplified further, but this is where i encountered the bug, if i pass the value as a string and parse it on the kotlin side it works.

Probably related to additionally borrowing a [JValue](https://github.com/jni-rs/jni-rs/blob/v0.21.1/src/wrapper/objects/jvalue.rs#L41-L48).


**Expected behavior**

Numeric values should be passable to kotlin functions.


**Environment:**

- Dioxus version:  v0.7.5
- Rust version:   rustc 1.94.0
- OS info:         Linux 6.19.11-1-cachyos
- App platform:    android

**Questionnaire**

I would like to fix, but don't have a solution yet.

--- a/packages/manganis/manganis-macro/src/ffi.rs
+++ b/packages/manganis/manganis-macro/src/ffi.rs
@@ -502,31 +502,31 @@ impl FfiBridgeParser {
             let (binding, arg_expr) = match &arg.ty {
                 ForeignType::Bool => (
                     quote! { let #binding_name = manganis::jni::objects::JValue::Bool(if #name { 1 } else { 0 }); },
-                    quote! { #binding_name.borrow() },
+                    quote! { #binding_name },
                 ),
                 ForeignType::I8 | ForeignType::U8 => (
                     quote! { let #binding_name = manganis::jni::objects::JValue::Byte(#name as i8); },
-                    quote! { #binding_name.borrow() },
+                    quote! { #binding_name },
                 ),
                 ForeignType::I16 | ForeignType::U16 => (
                     quote! { let #binding_name = manganis::jni::objects::JValue::Short(#name as i16); },
-                    quote! { #binding_name.borrow() },
+                    quote! { #binding_name },
                 ),
                 ForeignType::I32 | ForeignType::U32 => (
                     quote! { let #binding_name = manganis::jni::objects::JValue::Int(#name as i32); },
-                    quote! { #binding_name.borrow() },
+                    quote! { #binding_name },
                 ),
                 ForeignType::I64 | ForeignType::U64 => (
                     quote! { let #binding_name = manganis::jni::objects::JValue::Long(#name as i64); },
-                    quote! { #binding_name.borrow() },
+                    quote! { #binding_name },
                 ),
                 ForeignType::F32 => (
                     quote! { let #binding_name = manganis::jni::objects::JValue::Float(#name); },
-                    quote! { #binding_name.borrow() },
+                    quote! { #binding_name },
                 ),
                 ForeignType::F64 => (
                     quote! { let #binding_name = manganis::jni::objects::JValue::Double(#name); },
-                    quote! { #binding_name.borrow() },
+                    quote! { #binding_name },
                 ),
                 ForeignType::String | ForeignType::StrRef => (
                     quote! {

